.github/workflows/build_test_push.yaml

```diff
@@ -92,15 +92,23 @@ jobs:
           push: false
           load: true
           file: Dockerfile.server
-          tags: "us.gcr.io/model-159019/gh:${{ github.sha }}-server-dev"
+          tags: "us-docker.pkg.dev/model-159019/router/gh:${{ github.sha }}-server-dev"
           build-args: |
-            BASE_IMAGE=us.gcr.io/model-159019/gh:${{ github.sha }}-dev
+            BASE_IMAGE=us-docker.pkg.dev/model-159019/router/gh:${{ github.sha }}-dev
 
       - name: Push server-dev image
         run: |
-          docker push us.gcr.io/model-159019/gh:${{ github.sha }}-server-dev
+          docker push us-docker.pkg.dev/model-159019/router/gh:${{ github.sha }}-server-dev
+
+      # TODO: remove step when full migration to Artifact Registry is complete.
```

Comment:
I actually don't think this is necessary since it's only used internally.

Reply:
Had a chat with Mark and this image tag is often used in development. so I'll keep it